modules/@apostrophecms/notification/ui/apos/apps/ApostropheNotification.js

```diff
@@ -0,0 +1,56 @@
+import Vue from 'apostrophe/vue';
+
+export default function() {
+  /* eslint-disable no-new */
+  return new Vue({
+    el: '#apos-notification',
+    data () {
+      return {
+        notifications: []
+      };
+    },
+    computed: {
+      apos() {
+        return window.apos;
+      }
+    },
+    async mounted() {
+      if (apos.scene !== 'apos') {
+        return;
+      }
+
+      this.notifications = await this.poll();
+      setInterval(async() => {
+        this.notifications = await this.poll();
+      }, 10000);
+    },
+    methods: {
+      async dismiss(notificationId) {
+        await apos.http.delete(`${apos.notification.action}/${notificationId}`, {});
+        this.notifications = await this.poll();
```

Comment:
Remove the deleted notification from the array, saving a double API hit.

Reply:
Done